src/main/java/com/fasterxml/jackson/databind/introspect/JacksonAnnotationIntrospector.java

```diff
@@ -307,6 +307,16 @@ public Boolean isIgnorableType(AnnotatedClass ac) {
         JsonIgnoreType ignore = _findAnnotation(ac, JsonIgnoreType.class);
         return (ignore == null) ? null : ignore.value();
     }
+
+    @Override
+    public JsonIncludeProperties.Value findPropertyInclusions(Annotated a)
+    {
+        JsonIncludeProperties v = _findAnnotation(a, JsonIncludeProperties.class);
+        if (v == null) {
+            return JsonIncludeProperties.Value.all();
```

Comment:
Should this not return `null`, since null checks are used in many places to indicate "no inclusions defined"?

Reply:
null check are done on the `included` field (the set of string describing the fields). `JsonIncludeProperties.Value.all().getInclude()` will return null, but we still want to represent the `all` notion.

Reply:
ahhh. That makes sense (although has some consequences on work wrt `AnnotationIntrospector.Pair`).